### ISSUE TITLE
Is there a way to plot velocities as arrows? 

### ISSUE THREAD

**Issue by swfwmd** [r-tmap/tmap#731]
Trying to figure out a way to plot map of velocities. Is there a way for the tm_lines  to plot arrow?

**Comment by mtennekes** [comment]
No, not yet, unless you make the arrow heads part of the lines. Is indeed a very nice feature, but has low priority for me since it takes some time to implement.

**Comment by Nowosad** [comment]
Closing in favor of https://github.com/r-tmap/tmap/issues/731
